commit 2456c8448042ad2d5d5e3ea3e0389d067d50c27e
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-06-25T17:24:36+01:00 (3 months ago)
    
    TRITON-1772 triton.git should reference 'joyent-repos' in repos.json

diff --git a/repos.json b/repos.json
index 8f058e8..d9fc431 100644
--- a/repos.json
+++ b/repos.json
@@ -99,6 +99,9 @@
                 "meta": true
             }
         },
+        {
+            "name": "joyent-repos"
+        },
         {
             "name": "jsstyle",
             "labels": {
